src/pybind/mgr/cephadm/tests/test_scheduling.py

```diff
@@ -240,6 +207,39 @@ def test_daemon_placement_match(dp, dd, result):
     assert dp.matches_daemon(dd) == result
 
 
+# * first match from the top wins
+# * where e=[], *=any
+#
+#       + list of known hosts available for scheduling (host_key)
+#       |   + hosts used for explict placement (explicit_key)
+#       |   |   + count
+#       |   |   | + section (host, label, pattern)
+#       |   |   | |     + expected result
+#       |   |   | |     |
+test_explicit_scheduler_results = [
```

Comment:
Are we just moving the location of these tests in the file or am I missing something?

Reply:
Yes, this table corresponds to the location where I put it (it was displaced with some new UT in the middle)